exporters/write_buffer.py

```diff
@@ -231,10 +230,13 @@ def get_key_from_item(self, item):
     def grouping_info(self):
         return self.items_group_files.grouping_info
 
-    def get_metadata(self, buffer_path, meta_key):
-        return self.metadata.get(buffer_path, {}).get(meta_key)
+    def get_metadata(self, key, module='write_buffer'):
```

Comment:
cool, seems we don't need at all the previous method signature, with two parameters, but if needed, I think it is better to add a different method, called get_metadata_for_file(). That will conserve compatibility with super class get_metadata() method and pair correctly with the existing set_metadata_for_file()